.gitignore

```diff
@@ -2,8 +2,10 @@ venv/
 __pycache__/
 ipynb_checkpoints
 glue/schema/*.parquet
+.pre-commit-config.yaml
```

Comment:
**[Thought, non-blocking]** After thinking about it for a bit, I think that I actually prefer the other side of the tradeoff described in the PR body: It strikes me as slightly friendlier to keep a local copy of the pre-commit config under version control, in spite of the duplication with the `ccao-data/actions/pre-commit` action, so that developers can run pre-commit locally with minimal overhead. Having to occasionally resolve conflicts between the local pre-commit config and the remote config in `ccao-data/actions/pre-commit` will sometimes be annoying, but I expect it will be a less common annoyance than requiring every developer to copy the remote config to their local repo before they start development (and then also update their local copy whenever the remote changes).

That being said, this is a weakly-held opinion; what do you two think?

Reply:
I agree with the sentiment. A question for me is: Given that we would include both in each repository, how valuable is it to transition to composite actions across all of our repositories? It doesn't seem clear to me that we would be significantly reducing lines of code or reducing the total number of files. 

What would positives from this be? I suppose we can have a single source of truth for which to look for all of our hook versions in the actions repo, and that at least upon PRs they are all standardized. Maybe this would be a good start for other actions functionalities I'm not aware of.

Reply:
Good point, there's definitely a tradeoff between DRY and developer experience here. I'm comfortable moving forward with the gitignore and just making sure we explain to the rest of the team how they'll be expected to manage local pre-commit configs going forward.

Reply:
See my response on the [actions PR](https://github.com/ccao-data/actions/pull/22#pullrequestreview-2006915791)